application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -81,7 +79,17 @@ private boolean wasThreadAlreadyHandled(long threadChannelId) {
     }
 
     private void handleHelpThreadCreated(ThreadChannel threadChannel) {
-        helper.writeHelpThreadToDatabase(threadChannel.getOwnerIdLong(), threadChannel);
+        threadChannel.retrieveMessageById(threadChannel.getIdLong()).queue(message -> {
+
+            long idAuthor = threadChannel.getOwnerIdLong();
+
+            if (isPostedByBot(message)) {
+                idAuthor = getAuthorFromPostByBot(message).getIdLong(); // check if bot otherwise
+                                                                        // returns author ID
```

Comment:
i think you may have misunderstood my intent of putting comments in earlier review here , they were just meant to sort of brief what method will do. Since now the method is pretty clear about the intent, we can safely drop these `//check if bot otherwise return author ID` comments.